doc/source/user_guide/io.rst

```diff
@@ -2521,6 +2521,18 @@ indicate missing values and the subsequent read cannot distinguish the intent.
 
 .. _Table Schema: https://specs.frictionlessdata.io/table-schema/
 
+.. note::
```

Comment:
this note is about impl. the guide is not about this, pls remove

Reply:
Removing this from here will make the note disappear, conflicting with this comment https://github.com/pandas-dev/pandas/pull/44722#issuecomment-984753867, I'll gladly remove after confirmation that the note should not be *anywhere*.

In the case that the note is supposed to be place somewhere, please let me know where.

Reply:
@jreback I'm don't think I agree with you here.  We're adding something to the table schema that is not specified in the schema specification, so that if someone wanted to use a non-pandas reader with our output, they should be aware of the effect of having this new field.

Reply:
I'd love to finish this PR, but I'd like @jreback and @Dr-Irv to work out what I should do. I have no clue if there is a chain of command or something but a concise answer to the question "where should I put the notes, if I should put them at all" will do

Reply:
@jmg-duarte Final say is up to @jreback, but I'd like him to respond to my comment here: https://github.com/pandas-dev/pandas/pull/44722#discussion_r766717340

Reply:
I think its *ok* to show something about the implementation format, just not right here. E.g. we could have *another* section where you lay it out in prose, or in the doc-string (or both). E.g. these are user docs, what you are writing is about the actual format layout. So happy to have that somewhere but it would have be complete and not just reference the new stuff.

So would be ok with adding this documentation as a followup.

@Dr-Irv lmk if that makes sense.

Reply:
> @Dr-Irv lmk if that makes sense.

Looking at the context in which `jmg-duarte` added this doc about the extension array, it seems that it fits.  Take a look at the final result here:  https://github.com/pandas-dev/pandas/blob/9d87d98fd4f62e293f0c764184c174713883d823/doc/source/user_guide/io.rst

He added this after descriptions of what happens with various pandas dtypes (`Period`, `Categorical`, etc.), so I do think his addition fits right in with that text.

Reply:
ok i c, maybe prefer that in the list above then e.g around L2470 or so (and not a note)